call.go

```diff
@@ -225,7 +225,6 @@ func (f *Func) callGraph(args *argBuilder) (
 			include := true
 			if args.filterInput != nil && !args.filterInput(value) {
 				log.Trace("excluding input due to failed filter", "value", value)
-				include = false
```

Comment:
Fixes:
```
call.go:228:5: ineffectual assignment to include (ineffassign)
                                include = false
```

The continue in line 228 ensures that the assignment to include in the loop body has no effect.